frontend/src/components/Table/SimpleTable.tsx

```diff
@@ -31,16 +41,32 @@ interface SimpleTableProps {
   rows: IRow[];
   sort?: (columnIndex: number) => ThProps['sort'];
   sortBy?: ISortBy;
+  stickyHeaders?: boolean;
```

Comment:
Maybe this name is a bit confusing? Because both options are using the table isStickyHeader, but the difference is the wrapper with a height and the inner scroll

Reply:
Yes...I thought about that...but the feature is sticky headers and the difference like you said is the "inner scroll" that provides have the header sticked